packages/client/hmi-client/src/views/Home.vue

```diff
@@ -30,7 +37,7 @@ const mockProjects: string[] = ['Mocked Project'];
 					<li v-if="key === Categories.Recents">
 						<NewProjectCard />
 					</li>
-					<li v-for="(project, index) in mockProjects" :key="index">
+					<li v-for="(project, index) in mockProjects" :key="index" @click="gotoModel()">
```

Comment:
Shouldn't clicking on the project card open project page? In that case you could leave this change out as it would be taken care of in https://github.com/DARPA-ASKEM/TERArium/pull/235

Reply:
removed